src/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+import HomePage from "./HomePage";
```

Comment:
Omit we want the default here for the lazy so the import will be from the original file ('pages/HomePage')